deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -517,6 +517,9 @@ export class StackComposer {
                 migrationConsoleEnableOSI: migrationConsoleEnableOSI,
                 migrationAPIEnabled: migrationAPIEnabled,
                 servicesYaml: servicesYaml,
+                // The default value is correct if we deploy the capture proxy (e.g. captureProxyServiceEnabled or captureProxyESServiceEnabled),
+                // but not if the user does it on their own (in which case we use captureProxySourceEndpoint)
+                sourceClusterEndpoint: !(captureProxyServiceEnabled || captureProxyESServiceEnabled) ? captureProxySourceEndpoint : undefined,
```

Comment:
I was a bit confused by this intially, but think I understand now, my hope is that this will clear up in future changes. What is the default value you mention here, is it the default with the migration console class?

Reply:
yeah, exactly, this one: `https://capture-proxy-es.migration.${props.stage}.local:9200` (this results from @AndreKurait 's service connect changes)